documentation/docs/04-hooks.md

```diff
@@ -95,10 +95,10 @@ type Response = {
 	body?: any;
 };
 
-type Handle<Context = any> = (
+type Handle<Context = any> = ({
```

Comment:
thanks for fixing this! I noticed it the other day as well, but forgot to update